### ISSUE TITLE
Class 'LayerGroup<P>' incorrectly extends base class 'Layer'

### ISSUE THREAD

**Issue by luismiguelprs** [geoman-io/leaflet-geoman#912]
Hi, I'm getting this error on angular build process using leaflet + leaflet-geoman-free.

```
ERROR in node_modules/@types/leaflet/index.d.ts:1087:14 - error TS2415: Class 'LayerGroup<P>' incorrectly extends base class 'Layer'.
  Types of property 'pm' are incompatible.
    Type 'PMLayerGroup' is missing the following properties from type 'PMLayer': getShape, enableRotate, disableRotate, rotateEnabled, and 8 more.

1087 export class LayerGroup<P = any> extends Layer {
                  ~~~~~~~~~~
node_modules/@geoman-io/leaflet-geoman-free/dist/leaflet-geoman.d.ts:32:15 - error TS2430: Interface 'LayerGroup<P>' incorrectly extends interface 'Layer'.

32     interface LayerGroup {
                 ~~~~~~~~~~
```

Package versions: 

"@geoman-io/leaflet-geoman-free": "^2.11.1"
"leaflet": "^1.7.1"
"@types/leaflet": "^1.7.0"

**Comment by Falke-Design** [comment]
Thanks for reporting this issue, we are on it.

@ryan-morris is there a way to fix this localy? A workaround until we release a new version.

**Comment by ryan-morris** [comment]
I had to kept my project pinned at version `2.10.0` for the same reason. Outside of that, adding `"skipLibCheck": true` to tsconfig.json under `compilerOptions`, but I believe that will apply to all `*.d.ts` in node_modules.

I did not see a quick option solution to the problem either. In a similar way to [#893](https://github.com/geoman-io/leaflet-geoman/issues/893) the `@types/leaflet` inheritance complicates making 2 different types for same property name on `Layer` and `LayerGroup` since `LayerGroup` inherits from `Layer`. I'm still looking into options though
